scipy/stats/_continuous_distns.py

```diff
@@ -5586,6 +5587,457 @@ def _stats(self, a, b, moments='mv'):
+            -0.00000000005155,
+            +0.00000000000629,
+            -0.00000000000020,
+            -0.00000000000009,
+            +0.00000000000002,
+        )
+        phi = np.polynomial.chebyshev.Chebyshev(bn, domain=(-1, 1))(x)
+        phi[(x >= 1) | (x < -1)] = float('NaN')
+        return phi
+
+    def phi_3(x):
+        """Calculates phi when 1 <= x < 4. If x is outside this range,
+        NaN value is returned."""
+        cn = (
+            +0.093910553905040,
+            -0.046051135650639,
+            +0.005514491616154,
+            +0.000033846759482,
+            -0.000179611135166,
+            +0.000047499284920,
+            -0.000007772699014,
+            +0.000000815142983,
+            -0.000000018823117,
+            -0.000000014734204,
+            +0.000000004204788,
+            -0.000000000732265,
+            +0.000000000091386,
+            -0.000000000007034,
+            -0.000000000000183,
+            +0.000000000000190,
+            -0.000000000000043,
+            +0.000000000000007,
+            -0.000000000000001,
+        )
+        phi = np.polynomial.chebyshev.Chebyshev(cn, domain=(1, 4))(x)
+        phi[(x >= 4) | (x < 1)] = float('NaN')
+        return phi
+
+    def phi_4(x):
+        """Calculates phi when 4 < x. If x is outside this range,
+        NaN value is returned."""
+        dn = (
+            +0.70366178131816,
+            -0.27450971486614,
+            +0.02625538561233,
+            +0.00204063338947,
+            -0.00173032587110,
+            +0.00052204968816,
+            -0.00011383083318,
+            +0.00001872991034,
+            -0.00000178872991,
+            -0.00000021703167,
+            +0.00000017781207,
+            -0.00000006150638,
+            +0.00000001614242,
+            -0.00000000348239,
+            +0.00000000060008,
+            -0.00000000006514,
+            -0.00000000000536,
+            +0.00000000000603,
+            -0.00000000000233,
+            +0.00000000000069,
+            -0.00000000000018,
+            +0.00000000000003,
+        )
+        w0 = np.real(sc.lambertw(np.exp(x)))
+        # For "very large values of x", sc.lambertw fails, instead the
+        # equation provided in the paper (see docstring) works better.
+        LIMIT_WHERE_TO_STOP_USING_LAMBERT_EQUATION_SOLVER = 400
+        large_x = x[x>LIMIT_WHERE_TO_STOP_USING_LAMBERT_EQUATION_SOLVER]
+        w0[x>LIMIT_WHERE_TO_STOP_USING_LAMBERT_EQUATION_SOLVER] = large_x-large_x*np.log(large_x)/(large_x+1)
+
+        w4 = 2.9262710624435
+        phi = np.polynomial.chebyshev.Chebyshev(dn)(2*w4/w0-1)/w0**2
+        phi[(x < 4)] = float('NaN')
+        return phi
+
+    x_is_iterable = hasattr(x, '__iter__')
+    if not x_is_iterable:
+        x = [x]
+    x = np.asarray(x)
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        phi = np.nansum(np.stack([f(x) for f in (phi_0, phi_1, phi_2, phi_3, phi_4)]), axis=0)
```

Comment:
Which warnings are filtered here?

Reply:
It filters `over`, `invalid` and `divide`. Now I made it more explicit using `numpy.errstate`.